src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -142,16 +143,13 @@ private StateUpdate ToStateUpdate(Delta delta)
 
         private Address GetAccountAddress(ByteString publicKeyByteString)
         {
-            if (publicKeyByteString == null || publicKeyByteString.IsEmpty)
-            {
-                return null;
-            }
+            if (publicKeyByteString == null || publicKeyByteString.IsEmpty) return null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)